docs/DeveloperGuide.md

```diff
@@ -320,6 +245,9 @@ Step 4. `UniqueReminderList` should also be updated daily with `ReminderSchedule
     * Pros: Reminders will always be up to date
     * Cons: Will require more processing power and might slow down the application
 
+### \[Proposed\] Data archiving
```

Comment:
Shall we remove this, since we are not going to write anything for this?

Reply:
Okay!